jQuery/ajax.js

```diff
@@ -6,6 +6,18 @@ function ajax(citizens) {
   populateSelectWithOptions("#character-select", characters);
   populateSelectWithOptions("#hometown-select", hometowns);
   populateSelectWithOptions("#size-select", sizes);
+
+  var selectHandler = function(event) {
+    var params = gatherParams();
+    var filteredCitizens = filter(params, event.data.citizens);
+    redrawContent(filteredCitizens);
+  };
+
+  $('#character-select').on('change', { citizens: citizens }, selectHandler);
+  $('#hometown-select').on('change', { citizens: citizens }, selectHandler);
+  $('#size-select').on('change', { citizens: citizens }, selectHandler);
+
+  selectHandler({data: { citizens: citizens }});
```

Comment:
I don't really like calling the handler manually like this for initial population of the content